### PR TITLE
fix: remove search query from url when empty

### LINKED ISSUE #3092: remove search query from url when empty
#### Step to reproduce:
- Go to Explore page
- toggle some filters
- `&search=` is in url even when the SearchBar is empty

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3092
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/rmrk/Gallery/Search/SearchBar.vue [hunk 1]
```diff
@@ -295,6 +295,7 @@ export default class SearchBar extends mixins(
   }
 
   public mounted(): void {
+    console.log('jarsen this.', this)
```
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/rmrk/Gallery/Search/SearchBar.vue]: sorry : )

**Review by roiLeo — APPROVED** [review]
✅ can't reproduce empty search, code lgtm

**Comment by prachi00** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 69.06 USD/KSM ~ 0.724 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x2cb8d676e9d4b723c230bc9ab544c0a49a2dbc28610f2f8cfe6cabd441af9285](https://kusama.subscan.io/extrinsic/0x2cb8d676e9d4b723c230bc9ab544c0a49a2dbc28610f2f8cfe6cabd441af9285)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**